### DISCUSSION TITLE
Can we read a parquet file from a remote server ( example from a python server as a Static file )?

### DISCUSSION THREAD

**Discussion by RajaPoseidon** [holdenmatt/duckdb-wasm-kit#17] [General]
Can we read a parquet file from a remote server ( example from a python server as a Static file )?

**Comment by holdenmatt** [comment]
Yes, I believe so.

e.g. if you go to https://shell.duckdb.org/ and type .examples, you can see some queries that do that. The same should work here via duckdb-wasm.
